### ISSUE TITLE
Is it only good with python 3?

### ISSUE THREAD

**Issue by xuzhang5788** [surmenok/keras_lr_finder#2]
When I use this function in python 2 in virtual environment, it couldn’t plot the figures correctly. The figure was just a vertical line at learning rate with 0.0001. Thanks

I tried your example in python 3 without in virtual environment, it worked well
